nucypher/policy/conditions/context.py

```diff
@@ -98,3 +100,21 @@ def get_context_value(context_variable: str, **context) -> Any:
         value = func(**context)  # required inputs here
 
     return value
+
+
+def _resolve_context_variable(input: Union[Any, List[Any]], **context):
+    if isinstance(input, list):
+        return [_resolve_context_variable(item, **context) for item in input]
+    elif is_context_variable(input):
+        return get_context_value(context_variable=input, **context)
+    else:
+        return input
+
+
+def resolve_any_context_variables(parameters: List[Any], return_value_test, **context):
```

Comment:
Looks great! Should we add some unit tests in `test_context` for this function?

Reply:
✅